### PR TITLE
Update versions

### LINKED ISSUE #27: Update plug-ins to current versions

### PR DESCRIPTION
**Pull request by robinvandenbogaard**
Fixes #27

Updated dependencies and plugins. Did not update the licence-maven-plugin because I reckon you do not want to depend on release candidates.
```
[INFO] The following plugin updates are available:
[INFO]   com.mycila:license-maven-plugin .................... 4.1 -> 4.2.rc3
```

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Awesome, thanks a lot, @robinvandenbogaard!

File: pom.xml [hunk 1]
```diff
@@ -168,7 +178,7 @@
           <plugin>
             <groupId>org.jreleaser</groupId>
             <artifactId>jreleaser-maven-plugin</artifactId>
-            <version>0.9.0</version>
+            <version>1.4.0</version>
```
Comment (gunnarmorling) [inline pom.xml:181]: @aalmiray, given this is a major version bump, will this require updates to the JReleaser config, too?

**Review by gunnarmorling — APPROVED** [review]
This LGTM, merging. Thank you so much, @robinvandenbogaard!

File: pom.xml (hunk 1 above)
Comment (aalmiray) [reply pom.xml:181]: I've tried my best to keep backward compatibility as much as possible

**Pull request merged**